src/m_menu.c

```diff
@@ -3205,8 +3212,6 @@ setup_menu_t stat_settings1[] =  // Status Bar and HUD Settings screen
   {"Armor Color Matches Type", S_YESNO, m_null, M_X, M_SPC, {"hud_armor_type"}},
   {"Smooth Health/Armor Count", S_YESNO, m_null, M_X, M_SPC, {"smooth_counts"}},
 
-  MI_RESET,
```

Comment:
Why did you delete the reset button?

Reply:
I noticed that reset to defaults doesn't properly call the "action" function associated with a menu item for some reason, those items don't reset correctly. I can add it back, but something is wrong with that feature currently.

Reply:
You're right, I hadn't even noticed. I guess we can remove the reset button from everywhere. Although it can be useful in Key Bindings menu for example. I will try to fix it.